package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "diia-inhouse",
   "scripts": {
-    "build": "npm run build --ws"
+    "build": "npm run build --ws --if-present"
```

Comment:
It seems that the `eslint-config` package does not have a build script. Similar cases will be ignored with the `--if-present` flag